client/classic/upe/payment-processing.js

```diff
@@ -207,14 +208,17 @@ async function createStripePaymentElement( api, paymentMethodType ) {
 
 	// Set the layout to accordion if OC is enabled.
 	if ( getStripeServerData()?.isOCEnabled ) {
+		const layout = {
+			type:
+				getBlocksConfiguration()?.OCLayout ||
```

Comment:
```suggestion
				getStripeServerData()?.OCLayout ||
```

As this is the classic checkout page.

I am getting the following console error and the payment element does not load.

<img width="895" height="323" alt="Screenshot 2025-09-16 at 12 11 12 PM" src="https://github.com/user-attachments/assets/d926cacb-312d-462c-aac6-86c2df1ece79" />

Reply:
Thanks, Mayisha! Suggestion applied. It works for me now 👍